### ISSUE TITLE
How to improve my plot

### ISSUE THREAD

**Issue by gsato-git** [gsato-git/Assignment-2#2]
I think I can make a function to make contourf, coastlines, and colorizing first, then I can apply the function inside the for loop.

**Comment by luisortizuriarte** [comment]
Yes. And this could make your code "cleaner" or easier to read. Not necessary but if you plan to do similar operations in the future could be good to keep a function stored somewhere accessible.
